mongodb/mongodb_installer.sh

```diff
@@ -434,38 +533,48 @@ echo
 
 CONFIG_FILE="/usr/local/copperegg/ucm-metrics/mongodb/config.yml"
 AGENT_FILE="/usr/local/copperegg/ucm-metrics/mongodb/mongodb.rb"
+CONFIG_TEMPLATE="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/config-template.yml"
 
 echo
-echo "Creating config.yml.  Press enter to use the default [in brackets]"
+echo "Creating config.yml."
 echo
+cp $CONFIG_TEMPLATE $CONFIG_FILE
 
-echo "copperegg:" > $CONFIG_FILE
-echo "  apikey: \"$API_KEY\"" >> $CONFIG_FILE
-echo "  apihost: \"$API_HOST\"" >> $CONFIG_FILE
-echo "  frequency: $FREQ" >> $CONFIG_FILE
-echo "  services:" >> $CONFIG_FILE
-echo "  - mongodb" >> $CONFIG_FILE
+sed -i "0,/API-KEY/s//$APIKEY/" $CONFIG_FILE
+sed -i "0,/FREQUENCY/s//$FREQ/" $CONFIG_FILE
+sed -i "0,/APIHOST/s//$API_HOST/" $CONFIG_FILE
 
+setup_admin_base_group "mongodb_admin" "MongoDB-Admin"
+setup_db_base_group "mongodb" "MongoDB"
 
-setup_base_group "mongodb" "mongodb"
 rc=1
 while [ $rc -ne 0 ]; do
     # loop with defaults until they get it right
-    echo "Configuring first mongodb server (required)"
+    echo "Configuring first MongoDB server (required)"
     setup_mongodb "`hostname | sed 's/ /_/g'`-mongodb" "localhost" "27017"
     rc=$?
```

Comment:
what if we had issue in configuring second DB and it gave this error ? we have replaced in config file for first database. But will still be configuring first server again.